src/plugins/intel_gpu/tests/test_cases/detection_output_test.cpp

```diff
@@ -397,6 +397,72 @@ class detection_output_test : public ::testing::Test {
         check_results(output_prim, 3, "-1 0 0 0 0 0 0");
     }
 
+    void test_forward_decrease_label_id() {
+        const bool share_location = true;
+        const int num_loc_classes = share_location ? 1 : this->num_classes;
+        const int keep_top_k = 5;
+        const int background_label_id = 0;
+        const float nms_threshold = 0.4f;
+        const int top_k = 5;
+        const float eta = 1.f;
+        const prior_box_code_type code_type = prior_box_code_type::corner;
+        const bool variance_encoded_in_target = false;
+        const float confidence_threshold = 0.02;
+        const int32_t prior_info_size = 4;
+        const int32_t prior_coordinates_offset = 0;
+        const bool prior_is_normalized = true;
+        const int32_t input_width = 1;
+        const int32_t input_height = 1;
+        const bool decrease_label_id = true;
+
+        auto& engine = get_test_engine();
+        cldnn::memory::ptr input_location = engine.allocate_memory({ type_to_data_type<T>::value, format::bfyx,{ this->num_of_images, this->num_priors * num_loc_classes * 4, 1, 1 } });
```

Comment:
Too long lines (419, 420), should be splitted into 2 lines. Also please add spaces after `format::bfyx, ` (related to lines 419, 420, 421)

Reply:
done